examples/llava/CMakeLists.txt

```diff
@@ -1,14 +1,22 @@
-set(TARGET clip)
-add_library(${TARGET} clip.cpp clip.h)
-install(TARGETS ${TARGET} LIBRARY)
-target_link_libraries(${TARGET} PRIVATE common ggml ${CMAKE_THREAD_LIBS_INIT})
+set(TARGET llava)
+
+add_library(${TARGET} STATIC llava.cpp llava.h clip.cpp clip.h)
```

Comment:
`STATIC` should be dropped here to allow building `llava` as a shared library.

Reply:
Yes, will update it to build a shared library as well.

Reply:
Great, from my tests that looks like the only change I need to integrate this with `llama-cpp-python`.

Reply:
Good, now it also builds as a shared lib.

Reply:
That works, thank you!

Ommiting the `STATIC` / `OBJECT` from `target_link_libraries` also seemed to work with just the `llava` target without the need for the `llava_shared` target but if having different targets is preffered I don't mind it.